src/components/Tour.vue

```diff
@@ -122,7 +167,16 @@ export default {
 	.v-step {
 		background-color: $mainColor;
 		color: white;
+		border: 2px solid white;
+	}
+	/*
+	// If the little arrow should have a border too:
```

Comment:
What's this? Cleaning this up would be good

Reply:
Sometimes I like to keep things around that took me a not-insignificant amount of time to figure out, even if it's currently not needed -- but I can just as well get rid of it :+1: